packages/toolkit/package.json

```diff
@@ -4352,6 +4352,7 @@
         "bytes": "^3.1.2",
         "cross-fetch": "^4.0.0",
         "cross-spawn": "^7.0.3",
+        "fast-deep-equal": "^3.1.3",
```

Comment:
Why is this needed?

Reply:
> and is only 13kB unpacked

All dependencies must be maintained, and that becomes a big problem over time. :) 13kB is a lot of code which may have bugs and CVEs...